base/bucket.go

```diff
@@ -112,6 +120,49 @@ func (spec BucketSpec) IsWalrusBucket() bool {
 	return strings.Contains(spec.Server, "walrus:")
 }
 
+func (spec BucketSpec) IsTLS() bool {
+	return strings.HasPrefix(spec.Server, "couchbases") || strings.HasPrefix(spec.Server, "https")
```

Comment:
Is gocb case sensitive as well?  If not, I think we should do a case-insenstive compare here

Reply:
gocb/gocbcore/gocbconnstr are all case-sensitive.